test/Test/Network/GRPC/MQTT/Message/Stream.hs

```diff
@@ -47,7 +47,7 @@ tests =
   testGroup
     "Network.GRPC.MQTT.Message.Stream"
     [ testTreeStreamWire
-    , testTreeStreamHandle
+    -- , testTreeStreamHandle
```

Comment:
As we discussed, we should uncomment this line and my comment here is just a reminder to do that.